packages/graphql-playground-middleware-hapi/src/index.ts

```diff
@@ -9,45 +9,40 @@ import {
 const pkg = require('../package.json')
 
 export interface Register {
-  (server: Server, options: MiddlewareOptions, next: any): void
-  attributes?: any
+  (server: Server, options: MiddlewareOptions): void
```

Comment:
Not too familiar with Typescript, so if I'm doing this wrong please let me know!